### PR TITLE
Added display possible figure movement. Closed #21

### LINKED ISSUE #21: Add display possible figure movement
Render new object was displayed possible figure movement.

### PR DESCRIPTION
**Pull request by artemqdh**

### CODE REVIEW

**Review by Kasyanov-Igor — DISMISSED** [review]

File: Checkers/Graphics/GraphicalGameField.cs [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 using Checkers.Logic;
 using Checkers.Logic.Interfaces;
+using System.Windows.Shapes;
 
 namespace Checkers.Graphics
 {
 	public class GraphicalGameField : Grid
 	{
-		private ICheckersInterface _gameField;
+		private ICheckersInterface _gameField;
```
Comment (TopAcademy2022) [inline Checkers/Graphics/GraphicalGameField.cs]: Remove space.

File: Checkers/Graphics/GraphicalGameField.cs [hunk 2]
```diff
@@ -53,11 +55,36 @@ public void RenderGameField()
...
+			// Render again
+			this.RenderGameField();
+		}
+
+		/*!
+		* @brief Showing possible figure movements.
+		 */
+		public void ShowPossibleMoves(List<byte[]> positions)
+		{
+			byte[,] gameField = this._gameField.GetGameFieldAsByteArray(); ///< Getting the Game Field
```
Comment (TopAcademy2022) [inline Checkers/Graphics/GraphicalGameField.cs]: Remove this line.

File: Checkers/Graphics/GraphicalGameField.cs [hunk 3]
```diff
@@ -53,11 +55,36 @@ public void RenderGameField()
...
+
+			foreach (byte[] position in positions) ///< Foreach through List
+			{
+				Rectangle rectangle = new Rectangle() ///< Creating the possible field movements
```
Comment (TopAcademy2022) [inline Checkers/Graphics/GraphicalGameField.cs]: Use [GraficalFigurePosibleMovement](https://github.com/TopAcademy2022/checkers/blob/main/Checkers/Graphics/GraficalFigurePosibleMovement.cs)

File: Checkers/Graphics/GraphicalGameField.cs [hunk 4]
```diff
@@ -53,11 +55,36 @@ public void RenderGameField()
...
+					Fill = new SolidColorBrush()
+					{
+						Color = Color.FromRgb(255, 0, 0) ///< Red rectangle
+					},
+					Opacity = 0.50 
+				};
+
+				this.Children.Add(rectangle);
+
+				Grid.SetRow(rectangle, position[1]);
```
Comment (TopAcademy2022) [inline Checkers/Graphics/GraphicalGameField.cs]: Nope, use position[0].

File: Checkers/Logic/CheckersInterface.cs [hunk 5]
```diff
@@ -28,5 +28,22 @@ public class CheckersInterface : ICheckersInterface
...
+		{
+			List<byte[]> result = new List<byte[]>();
+
+			foreach (KeyValuePair<char, int> pair in positions)
+			{
+				byte column = (byte)(pair.Key - 'A');
+
+				byte row = (byte)(8 - pair.Value);
+
+				byte[] array = { column, row };
```
Comment (TopAcademy2022) [inline Checkers/Logic/CheckersInterface.cs]: Rewrite. Row as start element.

**Review by TopAcademy2022 — CHANGES_REQUESTED** [review]

**Review by TopAcademy2022 — APPROVED** [review]

**Review by natalia21082000 — APPROVED** [review]

**Review by Kasyanov-Igor — APPROVED** [review]

**Review by Viktor080181 — APPROVED** [review]

**Pull request merged**